incubator/java-openliberty/stack.yaml

```diff
@@ -1,5 +1,5 @@
 name: Open Liberty
-version: 0.2.12
+version: 0.2.13
```

Comment:
Can we bump this to 0.2.14 so that 0.2.13 could be tied to https://github.com/appsody/stacks/pull/816/ ?

Reply:
Done :)